### PR TITLE
Document How to Exclude CXSDK when working with CAP.

### PR DESCRIPTION
**Pull request by MatKuhr**
## What Has Changed?

See title

### CODE REVIEW

File: docs/java/guides/cap-sdk-integration.mdx [hunk 1]
```diff
@@ -83,9 +83,19 @@ Add the respective SAP Cloud SDK dependencies to the POM file under the `srv` di
 <dependency>
     <groupId>com.sap.cloud.sdk.s4hana</groupId>
     <artifactId>s4hana-all</artifactId>
+    <exclusions>            
+        <exclusion>
+		    <groupId>com.sap.cloud.servicesdk</groupId>
+		    <artifactId>odata-v2-lib</artifactId>
+        </exclusion>
+    </exclusions>
```
Comment (Johannes-Schneider) [inline docs/java/guides/cap-sdk-integration.mdx:91]: Indentation seems to be off:

```suggestion
    <exclusions>            
        <exclusion>
            <groupId>com.sap.cloud.servicesdk</groupId>
	    <artifactId>odata-v2-lib</artifactId>
        </exclusion>
    </exclusions>
```

File: docs/java/troubleshooting.mdx [hunk 2]
```diff
@@ -215,3 +215,22 @@ When running `Destination#asHttp()` you encounter an exception message like `"De
   To solve that issue, please check every field in the destination configuration for valid input.
   Please find the official [documentation from the Destination Service](https://help.sap.com/viewer/cca91383641e40ffbe03bdc78f00f681/Cloud/en-US/42a0e6b966924f2e902090bdf435e1b2.html) for more details.
   The value for field "Token Service URL" may require the full service endpoint path, including e.g. `/oauth/token`.
+
+## Olingo Dependency Conflicts
+
+:::info **Symptoms**
+In your CAP based application you encounter exception messages like `NoSuchMethodError` or `ClassDefNotFoundException` on methods or classes of `org.apache.olingo.odata2`.
```
Comment (eugene-andreev) [inline docs/java/troubleshooting.mdx:222]: In some cases the feature is simply not working as expected.

Comment (eugene-andreev) [reply docs/java/troubleshooting.mdx:222]: Not sure how to rephrase this :/

**Review by Johannes-Schneider — COMMENTED** [review]
Otherwise LGTM.

File: docs/java/guides/cap-sdk-integration.mdx (hunk 1 above)
Comment (Johannes-Schneider) [reply docs/java/guides/cap-sdk-integration.mdx:91]: Nvm, when I checked the branches out locally, it looked fine.

File: docs/java/guides/cap-sdk-integration.mdx [hunk 3]
```diff
@@ -83,9 +83,19 @@ Add the respective SAP Cloud SDK dependencies to the POM file under the `srv` di
...
 </dependency>
 ```
 
+:::caution
+Please use the same exclusion also on any other OData V2 related dependencies from the SAP Cloud SDK you may be using.
+:::
+
```
Comment (artemkovalyov) [inline docs/java/guides/cap-sdk-integration.mdx:98]:
```suggestion
:::caution

Please use the same exclusion also on any other OData V2 related dependencies from the SAP Cloud SDK you may be using.

:::

```

**Review by artemkovalyov — APPROVED** [review]
LGTM. Should we make SXSDK and Olingo go with 4.0.

File: docs/java/troubleshooting.mdx (hunk 2 above)
Comment (MatKuhr) [reply docs/java/troubleshooting.mdx:222]: Okay, I'd say we take this as a starting point as possible symptoms & if we come across more examples we just add them :)

**Pull request merged**